PowerShell/JumpCloud Commands Gallery/Linux Commands/Linux - remote-assist - Installation script.md

```diff
@@ -25,15 +25,17 @@ fi
 set -u
 
 # set the RAA version string in the below variable before running the script.
-# the example format of version: v0.75.0
+# the example format of version: v0.172.0
```

Comment:
This would be nice automated. I can see customers being confused on what the current version is.